docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -534,17 +534,41 @@ Migrating Ansible content to a different collection
 
 First, look at `Ansible Collection Checklist <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
 
-To migrate content from one collection to another, you need to create three PRs as follows:
+To migrate content from one collection to another, if the collections are parts of `Ansible distribution <https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in>`_:
 
-#. Create a PR against the old collection to remove the content.
-#. Create a PR against the new collection to add the files removed in step 1.
-#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+#. Copy content from the source (old) collection to the target collection.
+#. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. The deprecation must be released after the copied content has been released in the target collection.
```

Comment:
```suggestion
#. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. The deprecation must be released after the copied content has been included in a release of the target collection.
```
I wonder if we have to make this even more precise: that it needs to be released in a version that will get included in the same Ansible z-stream than the collection where it's deprecated. The aim is to make sure that users can always make adjustments to avoid deprecation warnings.

Reply:
Done, we could merge this, if it's ok now. We could adjust the paragraph later.